plugin/src/main/java/org/opensearch/ml/processor/MLInferenceSearchResponse.java

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+package org.opensearch.ml.processor;
+
+import java.io.IOException;
+import java.util.Map;
+
+import org.opensearch.action.search.SearchResponse;
+import org.opensearch.action.search.SearchResponseSections;
+import org.opensearch.action.search.ShardSearchFailure;
+import org.opensearch.core.xcontent.XContentBuilder;
+
+public class MLInferenceSearchResponse extends SearchResponse {
```

Comment:
Not related to the PR, just curious why ML Inference processor is housed in the plugin directory and not in search-processor directory ?

Reply:
we had this discussion earlier, the main reason to host the ml inference processor in plugin folder is to get closers to the model, so that it's easier to get local model and remote model and compose their model inputs,